src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -169,6 +240,15 @@ public function configure(array $configuration): void
                 'str_contains',
             ]);
         }
+
+        if (PhpUnitTargetVersion::fulfills($this->configuration['target'], PhpUnitTargetVersion::VERSION_9_1)) {
+            $this->functions = array_merge($this->functions, [
```

Comment:
those are not native, global functions but methods of PHPUnit. please don't mix content of `->functions`.